src/app/lib/utilities/preprocessor/rules/topstories.js

```diff
@@ -20,8 +20,7 @@ const squashTopStories = jsonRaw => {
     groups = groups.filter(group => {
       if (squashKeys.includes(group.type)) {
         collectedItems = collectedItems.concat(group.items);
-
-        const foundStapline = pathOr(null, ['strapline', 'name'], group);
+        const foundStapline = pathOr(null, ['title'], group);
```

Comment:
Adding `strapline` to all `top-stories` is tedious. I have done this to allow me test the frontpage  top stories design. This should be removed though.